src/common/lib/python/common_extension.h

```diff
@@ -20,7 +20,13 @@ typedef struct {
 
 typedef struct {
   PyObject_HEAD
+  /** A pointer to the flatbuffer object that contains the task spec. The task
+   *  spec is inside of this. This is the pointer to free when we destruct the
+   *  PyTask object. */
+  uint8_t *task_reply;
```

Comment:
Addressed above (not necessary, I removed it at the cost of introducing a copy).